ably/options.go

```diff
@@ -31,7 +31,7 @@ type ClientOptions struct {
 	RealtimeHost string // optional; overwrite endpoint hostname for Realtime client
 	Environment  string // optional; prefixes both hostname with the environment string
 	Key          string // an authorization key in the 'name:secret' format
-	ClientID     string // optional;
+	ClientID     string // optional; required for realtime presence
```

Comment:
Not entirely true actually, but leave this in.  You can `enterClient` on behalf of another `clientId`.